resources/views/components/address.blade.php

```diff
@@ -6,37 +6,42 @@
 >
-    <div v-if="!isEmpty" {{ $attributes->whereDoesntStartWith('v-')->class('flex flex-col') }}>
-        @if ($check)
-            <template v-if="{{ $check }}">
-                <div class="bg-ct-primary absolute inset-y-0 left-0 w-1 rounded-l-xl"></div>
-                <x-heroicon-s-check class="text-ct-primary absolute right-5 top-7 w-5" />
-            </template>
-        @endif
-        <p class="text-base font-medium mb-2.5 pr-8">
-            @if($customTitle)
-                @lang($customTitle)
-            @else
-                <template v-if="billing && shipping">@lang('Shipping & billing address')</template>
-                <template v-else-if="shipping">@lang('Shipping address')</template>
-                <template v-else-if="billing">@lang('Billing address')</template>
-                <template v-else>@lang('Address')</template>
+    <div {{ $attributes->whereDoesntStartWith('v-')->class('flex flex-col') }}>
+        <template v-if="!isEmpty">
+            @if ($check)
+                <template v-if="{{ $check }}">
+                    <div class="bg-ct-primary absolute inset-y-0 left-0 w-1 rounded-l-xl"></div>
+                    <x-heroicon-s-check class="text-ct-primary absolute right-5 top-7 w-5" />
+                </template>
             @endif
-        </p>
-        <div class="flex flex-1 flex-wrap justify-between">
-            <ul class="flex flex-col gap-1">
-                <li v-if="company">@{{ company }}</li>
-                <li v-if="name">@{{ name }}</li>
-                <li v-if="street">@{{ street }}</li>
-                <li v-if="city">@{{ city }}</li>
-                <li v-if="country === 'NL'">@lang('The Netherlands')</li>
-                <li v-if="country !== 'NL'">@{{ country }}</li>
-            </ul>
-            @if (!empty($slot))
-                <div class="mt-auto flex flex-col self-end">
-                    {{ $slot }}
-                </div>
-            @endif
-        </div>
+            <p class="text-base font-medium mb-2.5 pr-8">
+                @if($customTitle)
+                    @lang($customTitle)
+                @else
+                    <template v-if="billing && shipping">@lang('Shipping & billing address')</template>
+                    <template v-else-if="shipping">@lang('Shipping address')</template>
+                    <template v-else-if="billing">@lang('Billing address')</template>
+                    <template v-else>@lang('Address')</template>
+                @endif
+            </p>
+            <div class="flex flex-1 flex-wrap justify-between">
+                <ul class="flex flex-col gap-1">
+                    <li v-if="company">@{{ company }}</li>
+                    <li v-if="name">@{{ name }}</li>
+                    <li v-if="street">@{{ street }}</li>
+                    <li v-if="city">@{{ city }}</li>
+                    <li v-if="country === 'NL'">@lang('The Netherlands')</li>
```

Comment:
Correct me if I'm wrong but this seems to be a bit weird to put into a package like this, why is this required?

Reply:
Correct but it was already there, if it needs to become something else please create a separated PR 😇

Reply:
I see, didn't know that it was already there 👍